mssw/MyDate/lib/MyDate.pm

```diff
@@ -0,0 +1,159 @@
+package MyDate;
+
+use 5.006;
+use strict;
+use warnings;
+
+use Carp qw( croak );
+use Data::Dumper;
```

Comment:
I am, and I cannot understate this, SERIOUSLY CHEEZED about this. :anguished: :feelsgood: